packages/core/src/views/connect/Sidebar.svelte

```diff
@@ -150,6 +156,13 @@
           {@html defaultBnIcon}
         {/if}
       </div>
+      {#if $_(`connect.${step}.sidebar.heading`, { default: '' })}
+        <h2 class="heading">
+          {$_(`connect.${step}.sidebar.heading`, {
+            default: heading
+          })}
+        </h2>
```

Comment:
We should change this element to a div. There's not really a reason for this to be an h2.

Reply:
We could also update the class name to "sidebar-heading" to be more specific.